### ISSUE TITLE
Improve logging

### ISSUE THREAD

**Issue by elenirotsides** [elenirotsides/Trivia-Bot#83]
**Is your feature request related to a problem? Please describe.**
We need a means of logging...we currently have nothing. I'd like to be able to track where and when problems occur, should they occur.

**Comment by centanomics** [comment]
Hi can i be assigned to this issue? I've built a few discord bots before and might be able to help you out.

Also where are you looking to log this information? In the console or somewhere else?

**Comment by elenirotsides** [comment]
Hi!! @centanomics I apologize for the delay in my response! I haven't been getting notifications for some reason and I need to look into my issue so I don't miss anymore messages. Yes of course! I just refactored the codebase though so definitely take a look and see if you're still interested in helping out. I'll assign you for now, let me know if you need anything!

I'm thinking maybe we can log info to the console, but if you have any ideas or other suggestions, feel free to let me know!

**Comment by centanomics** [comment]
I think starting out with just logging information to the console is good. I think an idea for later down the line could be email notifications? No idea what they would be used for though.

**Comment by elenirotsides** [comment]
Yeah I totally agree. The only use case for email notifications that I can think of is if something catastrophically fails. Logging in the console sounds like a good start!

**Comment by centanomics** [comment]
Awesome, I'll get started on that then!

**Comment by elenirotsides** [comment]
Hi @centanomics are you still working on this? Please feel free to reach out if you have any questions.

**Comment by centanomics** [comment]
Yes I am, apologies as I've been really busy lately. I do have the class method created I just need to get around to actually implementing it

**Comment by elenirotsides** [comment]
That's great! And no worries. Looking forward to your PR 😁
